traffic_ops/v3-client/server.go

```diff
@@ -117,6 +117,9 @@ func (to *Session) CreateServer(server tc.ServerNullable) (tc.Alerts, ReqInf, er
 	resp, remoteAddr, err := to.request(http.MethodPost, API_SERVERS, reqBody, nil)
 	reqInf.RemoteAddr = remoteAddr
 	if err != nil {
+		if strings.Contains(err.Error(), "400 Bad Request[400]") {
+			reqInf.StatusCode = 400
+		}
```

Comment:
We should just set `reqInf.StatusCode` to `resp.StatusCode` so that it's directly captured whatever value it happens to have, and doesn't rely on string pattern matching.

Reply:
Ok. Updated